docs/config/index.md

```diff
@@ -1711,17 +1711,11 @@ export default defineConfig({
 ```
 
 ::: tip
-To have a better type safety when using built-in providers, you can add one of these types (for provider that you are using) to your tsconfig's `compilerOptions.types` field:
+To have a better type safety when using built-in providers, you should reference one of these types (for provider that you are using) in your config file:
 
-```json
-{
-  "compilerOptions": {
-    "types": [
-      "@vitest/browser/providers/webdriverio",
-      "@vitest/browser/providers/playwright"
-    ]
-  }
-}
+```ts
```

Comment:
This implies `vitest.confit.ts`, right? Let's include the filename to disambiguate.

Reply:
This is part of the config documentation. The name can be different and is explained in a separate section. Added a link just in case for "config file"

Reply:
Wouldn't this assume single tsconfig setup where both `vite/vitest.config.ts` and test files are included in the same ts project?
There's a PR suggesting alternatives for global types https://github.com/vitest-dev/vitest/pull/6676. I'm not sure we need to explain too much tsc details though.

Reply:
The docs say add to a setup file or a config - the user can decide which one depending in their tsconfig setup. I think adding more here is unnecessary- if they have two tsconfigs, they know which one to add to